<launch>

    <rosparam param="/humans/face_identification/face_database_paths" subst_value="true" >
        - /home/ros/.pal/face_db/face_db.json
        - $(find hri_face_identification)/share/test_faces_db.json
    </rosparam>
    <rosparam param="/humans/face_identification/match_threshold">0.3</rosparam>
    <node name="hri_face_identification" pkg="hri_face_identification" type="identify" />

</launch>

